src/KafkaFlow.Abstractions/Consumers/IWorker.cs

```diff
@@ -17,5 +19,15 @@ public interface IWorker
         /// </summary>
         /// <param name="handler"><see cref="Action"/> to be executed</param>
         void OnTaskCompleted(Action handler);
```

Comment:
As discussed in the PR #438 ([this discussion](https://github.com/Farfetch/kafkaflow/pull/438#discussion_r1341949396)), should we make this an event also?